features/device-config/persistence/api/src/main/java/org/opennms/features/deviceconfig/persistence/api/DeviceConfigDao.java

```diff
@@ -32,9 +32,12 @@
 import org.opennms.netmgt.model.OnmsIpInterface;
 
 import java.util.List;
+import java.util.Optional;
 
 public interface DeviceConfigDao extends OnmsDao<DeviceConfig, Long> {
 
     List<DeviceConfig> findConfigsForInterfaceSortedByDate(OnmsIpInterface ipInterface);
```

Comment:
Do we need the `findConfigsForInterfaceSortedByDate` method?

Reply:
I will keep that for now.